_data/all_rules/games.json

```diff
@@ -0,0 +1,8 @@
+{
+    "title": "Gaming specific",
+    "rules": [
+        { "text": "If you're going to continue complaining about a particular game, then you must provide a considerable alternative or suffice to stop complaining." },
+        { "text": "If after more than one explanation and a video tutorial and you still do not comprehend a particular game, perhaps it's not the game for you." },
+        { "text": "If a game seems broken, more over consistently so, and by continued review it appears to be just only you experiencing this, maybe the game isn't for you." }
```

Comment:
what is in index.md has a slightly updated phrasing than this. Do you mind pulling them in?

Reply:
Sure, no problem